client/src/types/user.type.ts

```diff
@@ -1,15 +1,16 @@
 // 뷰 모델
 export interface IUser {
-    userId: string
-    userName: string
-    profileImg: string
-    email: string
+    userId: string;
+    userName: string;
+    profileImg: string;
+    email: string;
 }
 
 // 서버의 응답 모델
 export interface IUserModel {
-    user_id: string
-    user_name: string
-    profile_img: string
-    email: string
-}
\ No newline at end of file
+    // user_id: string
```

Comment:
```suggestion
    user_id: string
```
주석처리 하신 거 다시 살리고, 아래에 `_id: string;` 지우면 😎